pyproject.toml

```diff
@@ -0,0 +1,388 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "otf-addons-gcp"
+version = "v0.1.0"
```

Comment:
Can you change the version format to match OTF/the other addons? Bumpver should do it